### PR TITLE
#197 metafixes

### PR DESCRIPTION
**Pull request by baxgas**
@gammaw I tidied the meta stuff a bit and also inserted the play triggering of the document title > very curious if it can solve the iPhone lock screen display!!! Microsoft Edge seems to use the page title when audio is playing on lock screen...

### CODE REVIEW

File: app/components/RadioPlayer.vue [hunk 1]
```diff
@@ -368,17 +369,25 @@ export default {
...
+
       this.$store.commit('player/isArcsiPlaying', false)
       this.$store.commit('player/isStreamPlaying', true)
 
-      //Google Analytics 4 event: only send if it's a regular show on air
-      if(this.show_check){
+      // Google Analytics 4 event: only send if it's a regular show on air
+      if (this.show_check) {
         gtag('event', 'Radio play', {
-          'Show': this.show_title,
```
Comment (gammaw) [inline app/components/RadioPlayer.vue:377]: @baxgas any reason why you removed the apostrophes? Are you sure it would work without them?

**Review by gammaw — COMMENTED** [review]
@baxgas the changes look good, except one I didn't quite get (which seems unrelated with what the other changes are aiming at), can you please check (see inline comment)?

File: app/components/RadioPlayer.vue (hunk 1 above)
Comment (baxgas) [reply app/components/RadioPlayer.vue:377]: I think PHPstorm linter autofix might did it. It is ok without, sure it is a js object key, if it is a simple string it is correct like this

Comment (gammaw) [reply app/components/RadioPlayer.vue:377]: Ok, let's see.

**Review by gammaw — APPROVED** [review]

**Pull request merged**